assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -54,7 +54,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Merge with above . And use assert instead of print</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>